content/guide/avalanche9000-motivation.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: What is the Motivation behind the Avalanche9000 Upgrade?
+description: Learn about the largest network upgrade of Avalanche, that will lower the barrier to launch a blockchain and enable permissionless vlaidation of L1s.
+date: 2024-08-15
+authors: [martin_eckardt, owenwahlgren]
+topics: [Avalanche]
+---
+
+Avalanche9000 will be the largest network upgrade of Avalanche to date. This upgrade will introduce a new concept called L1s, which will replace the current Subnet model. In this article we will discuss the current limitations of Subnets, and how the changes suggested in [ACP-77](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/77-reinventing-subnets/README.md) will address these limitations in the Etna Upgrade.
+
+<YouTube id="eczkjrGG_As" />
+^^ placeholder
+
+## ACP-77
+
+### How Subnets Work Today
+
+![Subnets](/common-images/multi-chain-architecture/acp-77/subnets.png)
+
+The Avalanche network has a large set of validators who all participate in running and validating the Primary Network.
+
+The Primary Network consists of the C-Chain, P-Chain, and X-Chain, and is where most of the transactions on Avalanche are processed.
+
+In order to scale horizontally and offer more blockspace, a subset of validators can opt-in to validate an additional blockchain in parallel with the Primary Network, we call this a Subnet.
+
+### Subnets -> L1s
+With ACP-77 Subnets will become their own Layer 1 blockchain.
+
+![L1s](/common-images/multi-chain-architecture/acp-77/l1s.png)
+
+Instead of using subsets of validators from the Primary Network - we now use a new distinct set of validators.
+
+This change maintains the short time of finality and high throughput Avalanche Subnets are known for while also enabling a new set of features with options for completely customizable staking options, gas tokens, and governance.
+
+
+### Current Economic Barriers
+
+![Staking Requirements](/common-images/multi-chain-architecture/acp-77/subnets-staking-requirements.png)
+The Primary Network currently has a staking requirement of 2,000 AVAX.
+
+If we assume a price of $20 per AVAX, this would be a $40,000 staking requirement per validator.
+
+To launch a Subnet today, validators must meet the staking requirement for each subsequent Subnet they choose to validate. This is typically a native token to the Subnet.
+This model can quickly lead to a high economic barrier of entry in creating new Subnets as validators must meet the 2,000 AVAX requirement in addition to the staking requirements of any Subnets they want to validate.
```

Comment:
```suggestion
This model has created a significant economic barrier for the creation of new Subnets as validators must meet the 2,000 AVAX requirement, and a majority of Subnets prefer to run with a minimum of 5 validators.
```

No subnets in production or on fuji have ever required validators to stake, as many remained permissioned from the get go.